addons/event/views/event_registration_views.xml

```diff
@@ -335,4 +379,4 @@
             action="action_registration"
             groups="event.group_event_user"/>
     </data>
-</odoo>
\ No newline at end of file
+</odoo>
```

Comment:
Note that an end-of-file change is done in commit 2 while it should be done in commit 1, just for "perfect commits split" purpose :)